### PR TITLE
Changed the index page

### PR DESCRIPTION
**Pull request by ConstantinosGeorgiou**
Index modifications @georgioupanayiotis Can you please check this.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,58 +1,120 @@
...
-        #footer {
-            background-color:black;
-            color:white;
-            clear:both;
-            text-align:center;
-            padding:5px;
-        }
-    </style>
+    <meta charset="UTF-8">
+    <title></title>
```
Comment (georgioupanayiotis) [inline index.html]: The title should not be empty

Comment (georgioupanayiotis) [inline index.html]: You should also use meta tags inside head section 
```
<meta name="description" content="Free Web tutorials">
<meta name="keywords" content="HTML,CSS,XML,JavaScript">
<meta name="author" content="Hege Refsnes">
```

**Comment by georgioupanayiotis** [comment]
Make the links with the structure:
`<a href="http://www.w3schools.com/" target="_blank">Visit W3Schools!</a>
`
Target blank will open new tab

File: index.html (hunk 1 above)
Comment (ConstantinosGeorgiou) [inline index.html]: Done

**Pull request merged**